web/source/dom/touchAliasElement.ts

```diff
@@ -96,6 +96,8 @@ namespace com.keyman.dom {
     __caretTimerId: number;
     __activeCaret: boolean = false;
 
+    __midCaretSearching: boolean = false;
```

Comment:
Perhaps `__executingCaretSearch` is better? ('mid' is a bit confusing as it has mathematical meaning as well).